tests/eips/eip4844/conftest.py

```diff
@@ -0,0 +1,71 @@
+"""
+Pytest (plugin) definitions local to EIP-4844 tests.
+"""
+import pytest
+
+from ethereum_test_tools import (
+    Block,
+    TestPrivateKey2,
+    Transaction,
+    add_kzg_version,
+    to_address,
+    to_hash_bytes,
+)
+
+from .common import (
+    BLOB_COMMITMENT_VERSION_KZG,
+    TARGET_DATA_GAS_PER_BLOCK,
+    calc_excess_data_gas,
+    get_data_gasprice,
+)
+
+
+@pytest.fixture
+def block_intermediate(
+    parent_blobs: int,
```

Comment:
We could potentially check this value for zero:
- if it's greater than zero, return the block
- if it's zero, return `None`

We can then conditionally append the block to the blocks list, and make tests shorter (1 block tests for those that do not rely on the parent having blobs).

Reply:
I think we do this on line 69:
```python
        if parent_blobs != 0
        else [],
```
If parent blobs is 0 in this case we return []

Reply:
I've changed the above essentially:
```python
if parent_blobs != 0
        else None,
 ```
So then used this None logic in your last comment!!